src/main/java/org/isf/operation/manager/OperationBrowserManager.java

```diff
@@ -144,11 +150,39 @@ public boolean descriptionControl(String description, String typeCode) throws OH
 	 * Get the list of possible operation results
 	 * @return the found list
 	 */
-	public ArrayList<String> getResultsList() { //TODO (OP-372): deal with codes instead of translations!!!
-		ArrayList<String> resultsList = new ArrayList<String>();
-		resultsList.add(MessageBundle.getMessage("angal.operation.result.sucess"));
-		resultsList.add(MessageBundle.getMessage("angal.operation.result.failure"));
-		resultsList.add(MessageBundle.getMessage("angal.operation.result.undefined"));
-		return resultsList;
+	public LinkedHashMap<String, String> getResultsList() { 
+		if (resultsListHashMap == null) buildResultHashMap();
+		return resultsListHashMap;
+	}
+	
+	private void buildResultHashMap() {
+		resultsListHashMap = new LinkedHashMap<>();
+		resultsListHashMap.put("Success", MessageBundle.getMessage("angal.operation.result.success"));
+		resultsListHashMap.put("Failure", MessageBundle.getMessage("angal.operation.result.failure"));
+		resultsListHashMap.put("Unknown", MessageBundle.getMessage("angal.operation.result.undefined"));
```

Comment:
as "codes" in the DB I would put them lowercase

Reply:
> * let's use lowercase "keys" in the db
> * please resolve the conflicts
> * please apply imports standards (see README.md)

**All done.**

> in any case the migration script [step_57_multiple_operations_in_admission.sql](https://github.com/informatici/openhospital-core/blob/develop/mysql/db/step_57_multiple_operations_in_admission.sql) needs to be changed accordingly

script has been updated using lowercase **keys**